clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -296,6 +301,11 @@
                                         9 * 60 * 1000,
                                         Importance.MEDIUM,
                                         CommonClientConfigs.CONNECTIONS_MAX_IDLE_MS_DOC)
+                                .define(INTERCEPTOR_CLASSES_CONFIG,
+                                        Type.LIST,
+                                        "",
```

Comment:
@ijuma Because of changes in [KAFKA-2702](KAFKA-2702
ConfigDef toHtmlTable() sorts in a way that is a bit confusing) (read the discussion there for full background) if a config doesn't have a default its assumed to be required. All optional configs need some default, often resulting in null or "" as the default in many cases.